framework_tvb/tvb/core/services/project_service.py

```diff
@@ -36,10 +36,13 @@
 """
 
 import formencode
+import os
 
+from tvb.adapters.datatypes.db.time_series import TimeSeriesIndex
```

Comment:
This dependency is not good here (from 'core' towards 'adapters')

Reply:
True, I can check some attribute here that time series have and datatype measures don't, like 'data_ndim'.